app/controllers/species_list_controller.rb

```diff
@@ -485,7 +485,7 @@ def do_add_observations(species_list, query)
     # large species_lists, such as "Neotropical Fungi".
```

Comment:
@pellaea Does the comment about inefficiency refer to `species_list.observation_ids += ids`? 
If yes, we can get rid of it.

Reply:
Yes, exactly.  I wanted to leave that in there but commented out so we would know not to try it again.  Or at least be sure to benchmark it before adding it back in.  The alternative is ugly, but was way faster.